src/cwv/handler.js

```diff
@@ -25,19 +25,22 @@ const INTERVAL = 7; // days
 const auditType = Audit.AUDIT_TYPES.CWV;
 
 export async function CWVRunner(auditUrl, context, site) {
+  const deliveryConfig = site?.getDeliveryConfig() || {};
+  const dailyThreshold = deliveryConfig.dailyThreshold || DAILY_THRESHOLD;
+  const interval = deliveryConfig.interval || INTERVAL; // days
```

Comment:
This is needed only for testing on our demo sites where the traffic is not enough. The thresholds were decided based on that fact that for sites with low traffic, the revenue impact will not be enough and hence we do not consider low traffic pages.

Unless we have an agreement regarding the same, we should not merge this PR.

On a side note, there are multiple opportunities, each with their own threshold/intervals. If we go ahead with this PR we should 
* use opportunity specific thresholds (for instance `deliveryConfig.cwv.dailyThreshold`) and 
* ensure `INTERVAL` cannot be set to any arbitrary large value as that can impact fetching data from RUM.

Reply:
@vdua Thanks for the review! I’ve incorporated your feedback in the latest commit.
imho, for testing any opportunity end-to-end, I think it’s important to keep this configurable.